reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json

```diff
@@ -22,30 +22,33 @@
   } ],
   "licenses" : [ {
     "_id" : 0,
-    "id" : "LicenseRef-test-Apache-2.0-(multi-line)"
+    "id" : "GPL-2.0-only WITH Classpath-exception-2.0 AND BSD-3-Clause OR MIT"
```

Comment:
Instead of adding this as new id 0, how about adding it as id 10 to avoid shifting all other entries and keep the diff small (for a more pleasant review)?

Reply:
The IDs are generated by the deterministic serialization, thus there is no choice of IDs here.

Reply:
This list is intended to contain single license IDs, not SPDX expressions.

Reply:
> This list is intended to contain single license IDs, not SPDX expressions.

Why is that and why do you want to keep that? ...the list contains currently licenses after the license finding curations are applied.

Reply:
The purpose of the list is to reduce duplication which is less efficient if it can contain SPDX expressions.
As a standalone PR this change does not make sense to me, why don't you keep it together with the other commits? This would have helped me understanding what the plan behind this is.

Reply:
Let's make one large PR then. I had decided to split it up to give the reviewer the added confidence of an intermediate green CI build after test case changes.